patterns/explicit-construction.md

```diff
@@ -0,0 +1,46 @@
+##The Explicit Construction Anti-Pattern
+
+This is the most common anti-pattern. It is easy to fall into this when you haven't really grasped
```

Comment:
"This is a common practice anti-pattern" — it would be hard to back up the assertion that it's the "most common", since we're relying mostly on (lots of) anecdotal data here.